### PR TITLE
Adds golden tools for the CE

### LINKED ISSUE #18121: CE doesn't have anything to make him unique from his underlings
lets be honest the blueprints are useless for everything but fixing hull breaches with missing piping

he has no one up above the average engineers

### PR DESCRIPTION
**Pull request by Iamgoofball**
https://www.youtube.com/watch?v=CjWYbcbpiWA

:cl: Iamgoofball
rscadd: The CE got a pay raise and some better tools.
/:cl:

uses 1 token from #18170 
i dropped them to toolspeed 3 and reduced the max fuel

### CODE REVIEW

**Comment by Steelpoint** [comment]
>fixing your own bug report mere minutes after posting said report

if the CE wants his own unique tools then give him a special omni-tool that can swap to any of the basic tools on the fly.

**Comment by Bawhoppen** [comment]
@Steelpoint never dear god holy shit

**Comment by WJohn** [comment]
Creating non-issues and then making a "fix" PR literally seconds after.

The new meme is already born. Boy that was fast.

**Comment by MMMiracles** [comment]
@Bawhoppen i see literally no issue with the CE getting an omni-tool. if anything it'd be slower than swapping between tools in a belt but be a lot more compact.

not like compactness matters anyway when there's enough room in the internals box for a full toolset.

**Comment by optimumtact** [comment]
Shiny and Chrome :+1:

**Comment by optimumtact** [comment]
18:03 <%oranges> new rule
18:03 <%oranges> if you make a feature request as a issue report
18:03 <%oranges> and then make a pr fixing it
18:03 <%oranges> you get - 10 merge tokens

**Comment by Bawhoppen** [comment]
goof gonna get indicted postemptively

**Comment by Incoming5643** [comment]
Follow my pro strat instead: Code everything shoddily so it needs follow up fix pulls, and remain token positive foreeeeeeeeeeeeever

File: code/game/objects/items/weapons/tools.dm [hunk 1]
```diff
@@ -34,6 +34,13 @@
 	playsound(loc, 'sound/weapons/genhit.ogg', 50, 1, -1)
 	return (BRUTELOSS)
 
+/obj/item/weapon/wrench/golden
+	name = "golden wrench"
+	desc = "Destroying one of these makes fat neckbeards around the galaxy cry."
```
Comment (Incoming5643) [inline code/game/objects/items/weapons/tools.dm]: I'm glad we got this blazing team fortress 2 joke in while it was still fresh and relevant to the collective consciousness of _gamer culture_

Comment (Iamgoofball) [inline code/game/objects/items/weapons/tools.dm]: Are you implying its still not true?

**Comment by Razharas** [comment]
@Iamgoofball

File: code/game/objects/items/weapons/tools.dm (hunk 1 above)
Comment (Incoming5643) [inline code/game/objects/items/weapons/tools.dm]: No I'm just implying that it falls in that weird little crevasse of a game's existence where a game is fairly old but not yet "nostalgic".

**Comment by xxalpha** [comment]
Blatant and tasteless TF2 references. Delete this.

**Comment by lordpidey** [comment]
Let the multitool be golden too.

**Comment by GunHog** [comment]
Golden tools seem like a thing Gangs would have. Higher tech tools themed after his hardsuit (white and blue) seems more fitting.

**Comment by Cebasiz** [comment]
i occasionally play CE and this is trash.

it's ugly and garish. if you want to resprite CE tools to be cool looking fine but gold? yuck.

**Comment by Steelpoint** [comment]
Again, make a neat omni tool with all the basic tools in it.

**Comment by pudl** [comment]
I like the idea of an omni tool but maybe one that needs to be recharged with a cell charger or have its battery replaced? That sounds very engineer-y to me and i like the idea of it

File: code/game/objects/items/weapons/tools.dm (hunk 1 above)
Comment (KazeEspada) [inline code/game/objects/items/weapons/tools.dm]: Didn't you hear? Blizzard released Team Fortress 3.

**Comment by PKPenguin321** [comment]
>Golden tools seem like a thing Gangs would have.

I actually kind of like this idea, considering how often gangs will put their dominator behind 10,000 walls and airlocks

**Comment by Iamgoofball** [comment]
no

golden tools, for the CE to show off his swag and superior riches compared to his underlings

**Comment by PKPenguin321** [comment]
If you give gangs a toolkit you could have a golden toolbox with golden pimp insulated gloves and a golden welding helmet

**Comment by Supermichael777** [comment]
Gold "insulated" gloves

**Comment by pudl** [comment]
Goofballs insolence will spell the end of him

**Comment by Iamgoofball** [comment]
@Cebasiz I play CE all the time and find golden tools fucking hillarious and awesome

also I have sprites for it

File: code/game/objects/items/weapons/tools.dm [hunk 2]
```diff
@@ -422,6 +442,14 @@
 			user << "<span class='warning'>You need one rod to start building a flamethrower!</span>"
 			return
 
+/obj/item/weapon/weldingtool/golden
+	name = "golden welding tool"
+	desc = "Heat-proof gold is a rare and valuable asset."
+	icon_state = "g_welder"
+	item_state = "g_welder"
+	max_fuel = 420
```
Comment (Shadowlight213) [inline code/game/objects/items/weapons/tools.dm]: :confused:

**Comment by Shadowlight213** [comment]
The values are fucking INSANE. No. Just no.
420 fuel for the welder when the hugetank welder is 80 fuel?
4 times the speed for everything? This is just dumb. There is such a value jump here between this and the tier below that it's not even funny.

**Comment by Iamgoofball** [comment]
yes, they're the best of the best tools reserved for the best of the best engineer

I'm gonna make the toolset a steal objective too

**Comment by Bawhoppen** [comment]
goofo pls

**Comment by lordpidey** [comment]
I was against this until goof said he was adding a traitor objective for it.

**Comment by Bawhoppen** [comment]
why would syndies want some petty gold tho

**Comment by Iamgoofball** [comment]
@Bawhoppen all the other heads have "things to steal" that are actually useful to the head

reactive teleport armor, hypospray, captain's disk/laser, HoS's gun

the CE's blueprints are fucking useless let's be honest

**Comment by GunHog** [comment]
> why would syndies want some petty gold tho

Because they are somehow four times faster than normal tools, and the welder is obviously running a bluespace containment tank. (Btw the CE's boots are also an objective)

**Comment by Iamgoofball** [comment]
@GunHog CE's boots are just standard magboots with a fancy paint scheme. Same as the normal magboots that are literally everywhere.

**Comment by GunHog** [comment]
False. They have no slowdown.

**Comment by Iamgoofball** [comment]
which is negated by the fact that your hardsuit has slowdown, and in reality never comes into play because if you have a hardsuit and magboots, you're working in zero-grav enviroments/space where the standard movespeed stuff gets thrown completely out of whack

**Comment by optimumtact** [comment]
Goof I dont' mind a visual reskin but these buffs are rediculous and nobody will merge it in this state because it's bloody hard to claw back any buff without pissing off half the players.

**Comment by Iamgoofball** [comment]
I'm okay with nerfing them a bit, the point is mainly for the CE to actually have a one up on doing his job over the other engineers.

**Comment by xxalpha** [comment]
How about he has the best hardsuit and all access to engineering and all the engineering tools? Exactly what made you think these buffs were okay or that giving the CE gold tools like this is some korean MMO is okay?

**Comment by Iamgoofball** [comment]
>How about he has the best hardsuit and all access to engineering and all the engineering tools?
1. His hardsuit isnt anything special compared to the normal hardsuit? Maybe some extra rad protection but that rarely matters ever.
2. You realize all engineers have all access everywhere, right?
3. So do all the engineers?

> Exactly what made you think these buffs were okay or that giving the CE gold tools like this is some korean MMO is okay?

1. Because the CE has no actual benefit at doing his job over the other people in his staff.
2. Because gold is #swag as fuck

**Comment by PKPenguin321** [comment]
am i on the only one that actually needs the blueprints when i build autism forts? you need them to set the area so that you can actually make APCs and shit

**Comment by swankcookie** [comment]
I'm legitimately confused about this. Why would a head want shitty, soft-metaled bling to replace their tools.

**Comment by xxalpha** [comment]
If your solution to improving a role is giving it gold sprited versions of the same tools then quit game design college.

**Comment by Shadowlight213** [comment]
@xxalpha It's not a resprite. It's literally far, far better than even rnd and borg tools.

**Comment by GunHog** [comment]
Nerf the speeds, use a white and blue theme to the sprites (and make them look high tech), give the golden tool sprites to gangs!

**Comment by Iamgoofball** [comment]
@swankcookie thats the joke
@PKPenguin321 Yeah, but those aren't useful for your average engineering job of "fix broken shit"
@xxalpha no
@GunHog I'm not ditching the golden theme unless @WJohn is OK with that, and if so @WJohn i'll need a golden toolbox, golden gloves, and a golden welding mask to go with the bling set

I'll nerf the speeds though, I'm okay with that.

@Shadowlight213 There aren't any R&D "tools" in the conventional sense except for that one welder ATM.

**Comment by AdamElTablawy** [comment]
can i just comment that the CE already has two steal objectives and this is just an unnecessary third

honestly i'd say give him a rapid engineering device instead, which is a hybrid RCD/RPD/Airlock painter

**Comment by Iamgoofball** [comment]
Thats still not as useful as tools, you guys aren't getting the point

**Comment by CosmicScientist** [comment]
Great sprites... I dunno if they fit here... nor if they should be buffed... nor if they should reference dead things on addition.

**Comment by Shadowlight213** [comment]
@iamgoofball there's technically the took set implant which is kinda rnd tools.

**Comment by Iamgoofball** [comment]
They still dont work faster than normal tools

**Comment by Shadowlight213** [comment]
They do. They are borg tools which work twice as fast as normal tools.

**Comment by Iamgoofball** [comment]
Oh, there you go

Still though, CE deserves the best. I'll knock it down to toolspeed 3 though.

**Comment by Steelpoint** [comment]
The CE does deserve the best.

But this is just dumb looking, if your going to give the CE more advance tools then they should look like advance tools, not like someone made a tool out of solid gold, which is very heavy.

**Comment by xxalpha** [comment]
Add character preference for toolbelt or tool arm.

**Comment by Shadowlight213** [comment]
Ce starting with a tool arm would be really fucking neat tbh.

**Comment by Iamgoofball** [comment]
@xxalpha fuck off no more prefs
@Shadowlight213 action button clutter but I could put one in his locker too if he doesn't want to use the default tools
@Steelpoint thats kind of the joke here, best engineering tools made out of the worst materials

**Comment by PKPenguin321** [comment]
Wouldn't be hard to make a preloaded one-use autoimplanter

**Comment by vulpineryth** [comment]
Gold seems too mundane for this station though. What about bananairum with plasma core tools? Makes more sense then gold and just as pimping.
Also just give the CE a version of the experimental industrial welder. Works just as well and has benefits without giving CE speed Decon skills.

**Comment by Iamgoofball** [comment]
@vulpineryth you're missing the point

**Comment by Iamgoofball** [comment]
i dropped them to toolspeed 3 and reduced the max fuel

**Comment by TechnoAlchemisto** [comment]
these look dumb

**Comment by ChangelingRain** [comment]
I don't really like these sprites

**Comment by Iamgoofball** [comment]
@ChangelingRain well too bad our official sprite lead @WJohn made them

**Comment by TechnoAlchemisto** [comment]
they're not bad sprites, but gold tools look dumb.

**Comment by Iamgoofball** [comment]
too bad, it's my PR and my sprite request

**Comment by TechnoAlchemisto** [comment]
that's why im leaving feedback you dumb head

**Comment by Bawhoppen** [comment]
goof cmon man I thought you had learned

**Comment by ChangelingRain** [comment]
Imagine this but for the last three hours and it makes two links when he does it and one when the timer ends.

**Pull request closed without merging**